### DISCUSSION TITLE
自定义主题如何修改组件的背景颜色？

### DISCUSSION THREAD

**Discussion by liweiook** [element-plus/element-plus#17587] [Q&A]
![image](https://github.com/user-attachments/assets/8a44b482-6c3a-47bf-a014-e57834f4fa3d)
使用的是scss方式，颜色修改成功了，但是不知道组件背景颜色怎么修改。系统的主题色是淡蓝色，混入白色的组件很难看。

@forward 'element-plus/theme-chalk/src/common/var.scss' with (
  $colors: (
    'white': #ff0000,
    'black': #000000,
    'primary': (
      // 主色
      'base': #27ba9b,
    ),
  )
);

**Comment by liweiook** [comment]
算了，使用css变量去了，最起码控制台可以看到使用了什么变量，可以直接修改，scss方式的变量不明确。

**Reply by warmthsea** [reply]
你似乎可以这样（我不确定）
$组件名： {
    剩余类名：样式
}
